Rakefile

```diff
@@ -2,50 +2,50 @@
 # Rakefile for agent-payload
 #
 
-protoc_binary="protoc"
-protoc_version="3.5.1"
+protoc_legacy_version="3.5.1"
+protoc_legacy_binary="/tmp/protoc#{protoc_legacy_version}"
+
+protoc_version="21.12"
+protoc_binary="/tmp/protoc#{protoc_version}"
```

Comment:
nit: Lots of copy/pasted `/tmp` path prefixes but not a blocker

Reply:
Gonna leave this as my next PR will update it